python/Ganga/GPIDev/Lib/File/OutputFileManager.py

```diff
@@ -211,52 +196,41 @@ def getWNCodeForDownloadingInputFiles(job, indent):
 
     from Ganga.GPIDev.Lib.Dataset.GangaDataset import GangaDataset
     if job.inputfiles is None or len(job.inputfiles) == 0 and\
-            (not job.inputdata or ((not isType(job.inputdata, GangaDataset)) or\
-                not job.inputdata.treat_as_inputfiles )):
-        return ""
+            not (job.inputdata or (isinstance(job.inputdata, GangaDataset) and\
```

Comment:
Line was already broken up. Copying code as it was, was not enforcing a re-write, will ammend